app/tests/api/NewsTestCest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+class NewsTestCest
+{
+    public function RetrieveAllNewsViaAPI(ApiTester $I)
+    {
+        $I->haveHttpHeader('Content-Type', 'application/json');
+        $I->haveHttpHeader('Authorization', 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwOlwvXC9leGFtcGxlLm9yZyIsInVzZXIiOnsiaWQiOiIxIiwidXNlcl9uYW1lIjoiamVmZmkifSwiaWF0IjoxNjA1MzA1NzUxfQ.71WUCmFy0RoIFv8zryFMLs_zuSTp0-1VQdbXvnUIq-s');
```

Comment:
@jjsimonAB in test have a function that does the login and pass the jwt , never have it hardcoded since test will fail